warrant/aws_srp.py

```diff
@@ -176,12 +176,9 @@ def process_challenge(self, challenge_parameters, test_timestamp=None):
         user_id_for_srp = challenge_parameters['USER_ID_FOR_SRP']
         salt_hex = challenge_parameters['SALT']
         srp_b_hex = challenge_parameters['SRP_B']
-        secret_block_b64 = challenge_parameters['SECRET_BLOCK']
-        if sys.platform.startswith('win'):
-            format_string = "%a %b %#d %H:%M:%S UTC %Y"
-        else:
-            format_string = "%a %b %-d %H:%M:%S UTC %Y"
-        timestamp = test_timestamp or datetime.datetime.utcnow().strftime(format_string)
+        secret_block_b64 = challenge_parameters['SECRET_BLOCK'] 
+        timestamp = test_timestamp or datetime.datetime.utcnow().strftime("%a %b %d %H:%M:%S UTC %Y")
+        re.sub(r" 0(\d) ", r" \1 ", timestamp, 1)
```

Comment:
This should be assigned to a parameter and should only be done if `test_timestamp` was not passed in.

Not sure if this will be accepted, but if so, let's add a comment linking back to the issue.